doc/source/examples/constructing_universe.ipynb

```diff
@@ -37,7 +37,7 @@
     {
```

Comment:
<p>No output showing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>re-pinging this, is the lack of output here expected? (or is it ReviewNB being silly?)</p>